README.md

```diff
@@ -51,6 +51,7 @@ gem install transmission-rss-*.gem
 ```sh
 docker run -t \
   -v $(pwd)/transmission-rss.conf:/etc/transmission-rss.conf \
+  -v $(pwd)/transmission-rss:/home/ruby/.config \
```

Comment:
I get the following error running the updated command:

```
/usr/local/lib/ruby/2.5.0/fileutils.rb:232:in `mkdir': Permission denied @ dir_s_mkdir - /home/ruby/.conf
ig/transmission (Errno::EACCES)
```

Did it work for you locally?

Please also update the image version to `v1.2.0`.

Reply:
I'm not using the command per se, only in `docker-compose`. I assume it's because the directories have different owners.